verticles/graphql/src/main/java/com/gentics/mesh/graphql/filter/NodeFilter.java

```diff
@@ -42,8 +42,12 @@ private NodeFilter(GraphQLContext context) {
 			.add(new MappedFilter<>("schema", "Filters by schema", SchemaFilter.filter(context), content -> content.getNode().getSchemaContainer()));
 		filters.add(new MappedFilter<>("created", "Filters by node creation timestamp", DateFilter.filter(),
 			content -> content.getNode().getCreationTimestamp()));
+		filters.add(new MappedFilter<>("creator", "Filters by creator", UserFilter.filter(),
+			content -> content.getNode().getCreator()));
 		filters.add(new MappedFilter<>("edited", "Filters by node update timestamp", DateFilter.filter(),
 			content -> content.getContainer().getLastEditedTimestamp()));
```

Comment:
Check whether it may be possible that getContainer() could return null. Best to trace the getContainer calls and check whether any null handling is already in place somewhere in the code.